src/gui/FlashcardModal.tsx

```diff
@@ -37,55 +40,82 @@ export class FlashcardModal extends Modal {
     ) {
         super(app);
 
+        // Init properties
         this.plugin = plugin;
         this.settings = settings;
         this.reviewSequencer = reviewSequencer;
         this.reviewMode = reviewMode;
 
+        // Setup base containers
         this.modalEl.style.height = this.settings.flashcardHeightPercentage + "%";
         this.modalEl.style.width = this.settings.flashcardWidthPercentage + "%";
-        this.modalEl.addClass("sr-modal");
+        this.modalEl.setAttribute("id", "sr-modal");
+
+        this.contentEl.addClass("sr-modal-content");
+        // if (Platform.isMobile) {
+        //     this.contentEl.style.display = "block";
+        // }
```

Comment:
we can delete this if it's no longer required

Reply:
Yes, I must have not seen it while deleting the commented out code. Its now gone.